pmdarima/preprocessing/exog/fourier.py

```diff
@@ -154,8 +154,9 @@ def transform(self, y, exogenous=None, n_periods=0, **_):
 
         if n_periods and exog is not None:
             if n_periods != exog.shape[0]:
-                raise ValueError("If n_periods and exog are specified, "
-                                 "n_periods must match dims of exogenous")
+                raise ValueError(f"If n_periods and exog are specified, "
+                                 f"n_periods must match dims of exogenous "
+                                 f"({n_periods} != {exog.shape[0]})")
 
```

Comment:
The builds are failing because f-strings are not supports until 3.6

Reply:
Yes I know, I fixed it ;)